pkg/collector/corechecks/oracle/common/constants.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package common
+
+// IntegrationName is the name of the integration.
+const IntegrationName = "oracle"
+const IntegrationNameScheduler = "oracle-dbm"
```

Comment:
A part of Oracle monitoring is still running in Python as an integration check. This would mess up the scheduling if both modules had the same name. At some point, we will migrate the existing Python functionality to G. That will resolve the conflict. The reason for such a schedule is that we want to invest our time first in providing more value to the customers instead of migrating the existing functionality to another programming language.

Reply:
Would it be possible to rename the directory to match the name of the check?

Also, maybe a comment about python check here would make sense too.

Reply:
We plan to migrate the existing Python functionality here anyway. Then, we would also rename the integration name from `oracle-dbm` to `oracle`. Should I nevertheless rename the directory to `oracle-dbm` now?

Reply:
I think it is better to rename the directory now, to avoid confusion in the interim. Directory can be renamed later again easily if the integration changes name.

Reply:
Renamed it to `oracle-dbm`.